easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -85,205 +80,220 @@ public static boolean hasPermissions(@NonNull Context context, @NonNull String..
     }
 
     /**
-     * Request a set of permissions, showing rationale if the system requests it.
+     * Request a set of permissions, showing a rationale if the system requests it.
      *
-     * @param object
-     *         Activity or Fragment requesting permissions. Should implement
-     *         {@link android.support.v4.app.ActivityCompat.OnRequestPermissionsResultCallback}
-     *         or {@code android.support.v13.app.FragmentCompat.OnRequestPermissionsResultCallback}
-     * @param rationale
-     *         a message explaining why the application needs this set of permissions, will be displayed if the user rejects the request the first
-     *         time.
-     * @param requestCode
-     *         request code to track this request, must be < 256.
-     * @param perms
-     *         a set of permissions to be requested.
+     * @param activity    Activity requesting permissions. Should implement {@link
```

Comment:
Since we're now going to have a huge number of methods and overloads (which is good!) this file is going to be overloaded with duplicate docs.

In general, for overloads we should avoid duplicating the whole docstring and just say something short and then use {@see} to point to the method we delegate to which contains the full docstring.

"Requests permissions with default values for rationale dialog buttons.

{@see #requestPermissions(...)}"